doc/newfeat.texi

```diff
@@ -4,6 +4,18 @@
 Changes and New Features in development version:
 @itemize @bullet
 
+@item @ESS{[R]}: The R package mode now sets @code{ess-dialect} to
```

Comment:
This probably shouldn't be in the news. It's an internal change. Or can be something more generic "Improved support for ess commands outside ess modes".